### PR TITLE
fix bug scrolling

### PR DESCRIPTION
**Pull request by KhRania**
## What does this do?

fix bug scrolling in featured products

## Why did you do this?

fix bug scrolling in featured products

 

## Who/what does this impact?

User Experience

## How did you test this?

 
localhost

### CODE REVIEW

File: src/component/Main/index.jsx [hunk 1]
```diff
@@ -206,7 +213,7 @@ const Main = () => {
                 <CourseTrainingCard
                   id={training._id}
                   key={training._id}
-                  thumbnail={training.Thumbnail.filePath}
+                  thumbnail="./images/home/right.png"
```
Comment (TomIA24) [inline src/component/Main/index.jsx:216]: why changed this ?

**Review by TomIA24 — CHANGES_REQUESTED** [review]

File: src/component/Main/index.jsx (hunk 1 above)
Comment (KhRania) [reply src/component/Main/index.jsx:216]: My bad filepath return a big on localhost so I changed with fix image

Comment (elayeboussama) [reply src/component/Main/index.jsx:216]: Create a conditional check: if training.thumbnail.filepath exists, display it; otherwise, show a default image

Comment (KhRania) [reply src/component/Main/index.jsx]: done

**Review by elayeboussama — APPROVED** [review]

**Pull request merged**